starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,53 @@
 // Names and Input
+var hacker1 =  'Pablo';
+console.log(`The Driver's name is ${hacker1}`);
 
+var hacker2 = window.prompt('Enter navigator\'s name');
```

Comment:
Just want to make note that window.prompt will throw an error unless the JS file is linked to an HTML document. Not a huge focus for this activity, but something to be aware of.